src/intersection-observer-polyfill.js

```diff
@@ -375,16 +413,51 @@ export class IntersectionObserverPolyfill {
     changeEntry.target = element;
     return changeEntry;
   }
+
+  /**
+   * Handle Mutation Oberserver events
+   * @private
+   */
+  handleMutationObserverNotification_() {
+    if (this.mutationPass_.isPending()) {
+      return;
+    }
+
+    // Wait one animation frame so that other mutations may arrive.
+    this.mutationPass_.schedule(16);
```

Comment:
@torch2424 @jridgewell if the MutationObserver API already batch events to animation frames, why do we need this delay here?

Reply:
From my testing, I noticed that Mutation Observer is batched, though I trust @jridgewell 's judgement on the issue that we should use this.

Reply:
The original design for `FixedLayer` had all "updates" happening behind the pass. This would queue up the request when a sync request came in, or when an async mutation was detected. But, there were a bunch of testing changes that would have been required, so I undid the sync changes.

I can't think of a technical reason to need this anymore.